2024-03-04.mdx

```diff
@@ -0,0 +1,13 @@
+---
+date: 3-4-2024
+---
+
+### What's new
+
+- We have added a way to disable search on your database. The reason you would want to do that is cost: if you don't need any of the search functionality (search, aggregation, vector search), you don't need to pay the storage costs in Elasticsearch. You can find the toggle in the database settings. Don't worry though, search is still included in the free tier and enabled by default.
+
+  ![Search toggle in database settings](images/search_toggle.png)
+
+- In preparation for our launch week (shhh! it's coming soon) we have re-structured our docs to make it easier to find what you're looking for. Please let us know if you have any feedback! Also quick reminder: the search bar in the docs works really well (because it's basd on Xata).
```

Comment:
Let's not give away all our secrets 😃

Fixed a typo too.

```suggestion
- In preparation for our upcoming releases, we have re-structured our docs to make it easier to find what you're looking for. Please let us know if you have any feedback! Also quick reminder: the search bar in the docs works really well (because it's based on Xata).
```